temoa/model_checking/unit_checking/relations_checker.py

```
@@ -0,0 +1,518 @@
+            query = f'SELECT {tech_column}, units, NULL FROM {table_name}'
+        case RelationType.COMMODITY:
+            query = f'SELECT commodity, units, NULL FROM {table_name}'
+        case _:
+            raise ValueError(f'Unexpected relation type: {relation_type}')
+
+    try:
+        rows = conn.execute(query).fetchall()
+    except sqlite3.OperationalError as e:
+        # Log the error but don't fail the entire check
+        logger.exception(
+            'failed to process query: %s when processing table %s: %s', query, table_name, e
```

Ruff (0.14.7)

Redundant exception object included in `logging.exception` call

(TRY401)